templates/rustdoc/topbar.html

```diff
@@ -210,7 +210,9 @@
     </li>{#
 
     Display the platforms that the release has been built for
-    #}<li class="pure-menu-item pure-menu-has-children">
+    #}
+    {% if metadata.doc_targets %}
```

Comment:
```suggestion
    {%- if metadata.doc_targets -%}
```

IIRC it's important to avoid whitespace here to avoid extra spacing between each menu element.